src/ResourceManager/KeyVault/Commands.KeyVault/Commands/GetAzureKeyVaultCertificate.cs

```diff
@@ -146,5 +171,20 @@ private void GetAndWriteCertificatesVersions(string vaultName, string name, stri
                 WriteObject(pageResults, true);
             } while (!string.IsNullOrEmpty(options.NextLink));
         }
+
+        private void GetAndWriteDeletedCertificates( string vaultName )
+        {
+            KeyVaultObjectFilterOptions options = new KeyVaultObjectFilterOptions
+            {
+                VaultName = VaultName,
+                NextLink = null
+            };
+
+            do
+            {
+                var pageResults = DataServiceClient.GetDeletedCertificates(options);
```

Comment:
This follows the existing pattern we use for keys and secrets. The reason we print this one page at a time is to be responsive. This operation could take a long time (and even fail/time out) when the number of versions is high.